pkgs/development/python-modules/graphite-web/default.nix

```diff
@@ -15,8 +28,19 @@ buildPythonPackage rec {
   ];
 
   propagatedBuildInputs = [
-    django django_tagging whisper pycairo cairocffi
-    ldap memcached pytz urllib3 scandir
```

Comment:
txamqp, gunicorn, pyparsing, whitenoise and six where addded.

Reply:
Yes, they are required according to https://github.com/graphite-project/graphite-web/blob/1.1.7/requirements.txt.